pylcm/__init__.py

```diff
@@ -55,7 +61,7 @@ def msg_getconstants (lcm_msg):
     return constantslist
 
 
-def msg_to_dict (data, e_channel, msg, statusMsg, verbose=False, lcm_timestamp=-1):
```

Comment:
could you set up black, flake8 and pylint as pre-commit hooks for this repo as done in `dexai-python`? there's lot of styling issues

Reply:
I'll do this now.